crates/ruff/tests/lint.rs

```diff
@@ -510,6 +510,307 @@ ignore = ["D203", "D212"]
     Ok(())
 }
 
+#[test]
+fn nonexistent_config_file() {
+    assert_cmd_snapshot!(Command::new(get_cargo_bin(BIN_NAME))
+        .args(STDIN_BASE_OPTIONS)
+        .args(["--config", "foo.toml", "."]), @r###"
+        success: false
+        exit_code: 2
+        ----- stdout -----
+
+        ----- stderr -----
+        error: invalid value 'foo.toml' for '--config <CONFIG_OPTION>'
+
+          tip: The `--config` flag must either be a path to a `.toml` configuration file or a TOML `<KEY> = <VALUE>` pair overriding a specific config setting
+          tip: The path `foo.toml` does not exist on your filesystem
```

Comment:
Maybe this should be prefaced with.. "It looks like you passed a configuration file, but the path ... does not exist."

"on your filesystem" feels redundant.

Reply:
Or to be less "you": "It looks like a configuration file was passed, but..."